src/Timeline.ts

```diff
@@ -290,6 +290,10 @@ export default abstract class Timeline extends Component{
             (resolve => canvas.toBlob(resolve, 'image/png')),
         )!;
 
+        if (blob === null) {
```

Comment:
In which case, blob will be `null`?

Reply:
From MDN docs [here](https://developer.mozilla.org/en-US/docs/Web/API/HTMLCanvasElement/toBlob#parameters)

> A callback function with the resulting [Blob](https://developer.mozilla.org/en-US/docs/Web/API/Blob) object as a single argument. null may be passed if the image cannot be created for any reason.

So typescript was complaining.